core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -1911,10 +1911,17 @@ class LogTest {
     val bogusIndex2 = Log.offsetIndexFile(logDir, 5)
     val bogusTimeIndex2 = Log.timeIndexFile(logDir, 5)
 
+    // Create index files
```

Comment:
Probably add a comment why we need to explicitly create those files.

Reply:
Done.